package.json

```diff
@@ -45,7 +46,8 @@
     "simple-peer": "9.11",
     "stream-to-it": "^0.2.3",
     "webrtc-stun": "3.0.0",
-    "wrtc": "0.4.7"
+    "wrtc": "0.4.7",
+    "yargs": "^17.0.1"
```

Comment:
`yargs` can be safely turned into a devDependency. No need to install `yargs` when importing `hopr-connect`